include/SFML/Audio/Music.hpp

```diff
@@ -134,6 +177,46 @@ class SFML_AUDIO_API Music : public SoundStream
     ////////////////////////////////////////////////////////////
     Time getDuration() const;
 
+    ////////////////////////////////////////////////////////////
+    /// \brief Get the positions of the of the sound's looping sequence
+    ///
+    /// \return Loop position class.
+    ///
+    /// \warning Since setLoopPoints() performs some adjustments on the
+    /// provided values and rounds them to internal samples, a call to
+    /// getLoopPoints() is not guaranteed to return the same times passed
+    /// into a previous call to setLoopPoints(). However, it is guaranteed
+    /// to return times that will map to the current internal samples of
+    /// this Music if they are later passed to setLoopPoints().
+    ///
+    /// \see setLoopPoints
+    ///
+    ////////////////////////////////////////////////////////////
+    LoopPoints getLoopPoints() const;
+
+    ////////////////////////////////////////////////////////////
+    /// \brief Sets the beginning and end of the sound's looping sequence using sf::Time
+    ///
+    /// Loop points allow one to specify a pair of positions such that, when the music
+    /// is enabled for looping, it will seamlessly seek to begin whenever it encounters end.
+    /// The input values are clamped to the duration of the sound. If they are the same,
+    /// then a closed loop cannot be formed, and this function will "reset" the loop to the
+    /// full length of the sound. Note that the implicit "loop points" from the end to the
+    /// beginning of the stream are still honored. Because of this, "reverse" loop ranges,
+    /// where end comes before begin, are allowed, and will cause the sound to loop everything
+    /// "outside" of the specified range. This function can be safely called at any point
+    /// after a stream is opened, and will be applied to a playing sound without affecting
+    /// the current playing offset.
+    ///
+    /// \param loopPts The definition of the loop. Can be any time points within the sound's length
+    ///
+    /// \return True if the times were valid and the operation succeeded, false otherwise
+    ///
+    /// \see getLoopPoints
+    ///
+    ////////////////////////////////////////////////////////////
+    bool setLoopPoints(LoopPoints loopPts);
```

Comment:
Returning bool as an error reporting mechanism is an idiom that SFML only uses for *runtime errors* (which can occur unexpectedly, even in correct programs, like a non-existing file). Particularly, it's the only setter that does this.

Using out-of-bounds loop points on the other hand is a *logic error*, a bug that doesn't occur if the caller is aware of his music's length (which can be easily polled). Since a user is very likely to ignore this return value under the assumption that the call succeeded, I would rather use a different error reporting mechanism (`sf::err()` or `assert`).